frontend/components/Navbar/Navbar.tsx

```diff
@@ -1,15 +1,34 @@
 import Image from 'next/image'
 import Link from 'next/link'
+import { useState } from 'react'
 import logo from '../../public/logo.png'
+import { Button } from '../Button/Button'
+import { IconButton } from '../Button/IconButton'
+
+const navLinks = [{
+    to: "/discover",
```

Comment:
```suggestion
    to: "/weights",
```

Das mal darauf setzten :)